source/MaterialXGraphEditor/RenderView.cpp

```diff
@@ -688,42 +621,11 @@ void RenderView::initContext(mx::GenContext& context)
     // Initialize search path
     context.registerSourceCodeSearchPath(_searchPath);
 
-    // Initialize color management.
```

Comment:
Moved out to where document setting is done to get the appropriate setup on new documents instead of just renderer initialization.